src/bin/run.js

```
@@ -38,13 +39,19 @@ program
     'Select the browser to run your tests on. E.G. chrome, firefox, etc.'
   )
   .option('c, --config [config]', 'Path to your config')
+  .option('--run [optional]', 'Filter scenarios based on label name')
   .option('r, --remote', 'Upload new baseline to remote storage')
   .action(async options => {
     try {
       const config = require(path.resolve(options.config)); // eslint-disable-line import/no-dynamic-require
 
       if (options.browser) config.browser = options.browser;
 
+      if (options.run)
```

I think we should validate the config before filtering.

`fail fast`